terminalcraft/app/page.tsx

```diff
@@ -1,290 +1,1220 @@
 "use client"
-import { useState, useEffect } from "react";
-import Prompt from "@/components/Prompt";
+import { useState, useEffect } from 'react';
+import { 
+  Rocket, 
+  AlertTriangle, 
+  X, 
+  CheckCircle, 
+  ExternalLink, 
+  Maximize2, 
+  Minimize2, 
+  Star, 
+  ChevronRight,
```

Comment:
please avoid auto-formatting your changes before committing. that makes it hard to review what actually changed.